### DISCUSSION TITLE
Comments character limit bypass

### DISCUSSION THREAD

**Discussion by thebruhmoment** [dayanch96/BHTikTok-Plus#25] [Ideas]
thanks for the tweak!
Ok so you just engaged in real discussion in tiktok (🤡) but tiktok wont allow long comments and only limits you to 150 characters. people have figured you can split your comment with 
1/4
2/4
3/4
4/4 in the start of the comment
so why not make this process automated?
make the feature remove that limit from the comment form, and after pressing "send" it will calculate total character count. and split the messages evenly. it also gotta make sure its not splitting the word in half when doing this so it doesent end up looking crooked. if the message was a response to somebody's else comment make it save that state for all split comments.
also would be nice if ,we say, your original comment was 160 characters it makes each comment 80, not 150 and 10. (for example)

**Comment by dayanch96** [comment]
Hello. Thanks - that's an interesting proposal, but unfortunately I won't be working on it. Busy with personal stuff lately. Other than that, posting more than 1 comment at the same time might be counted as spam and you can get shadow ban
